lib/logstash/outputs/opensearch/http_client_builder.rb

```diff
@@ -35,6 +35,7 @@ def self.build(logger, hosts, params)
       end
 
       common_options[:timeout] = params["timeout"] if params["timeout"]
```

Comment:
While we are here, does specifying timeout = nil make the request not timeout? For another PR/workitem I'm used to seeing `if params.key?(:timeout)` and support all values with predictable behavior.